packages/web-react/src/components/Tooltip/__tests__/Tooltip.test.tsx

```diff
@@ -29,3 +30,94 @@ describe('Tooltip', () => {
     expect(element.textContent).toBe('Hello World');
   });
 });
+
+describe('TooltipModern', () => {
+  stylePropsTest((props) => <TooltipModern {...props} data-testid="TooltipModern-test" />, 'TooltipModern-test');
+
+  restPropsTest((props) => <TooltipModern {...props} />, 'div');
+
+  const id = 'TooltipModernTest';
+  const triggerText = 'TooltipTrigger';
+  const popoverText = 'TooltipPopover';
+
+  it('should render tooltip', () => {
+    const onToggle = () => null;
+    const open = true;
+
+    const dom = render(
+      <TooltipModern id={id} isOpen={open} onToggle={onToggle}>
+        <TooltipTrigger>{triggerText}</TooltipTrigger>
+        <TooltipPopover>{popoverText}</TooltipPopover>
+      </TooltipModern>,
+    );
+
+    const triggerElement = dom.container.querySelector(`#${id}`) as HTMLElement;
+    const popoverElement = dom.container.querySelector('[data-spirit-element="tooltip"]') as HTMLElement;
+
+    expect(triggerElement.textContent).toBe(triggerText);
+    expect(popoverElement.textContent).toBe(popoverText);
+  });
+
+  it('should be opened', () => {
+    const onToggle = jest.fn();
+
+    const dom = render(
+      <TooltipModern id={id} isOpen onToggle={onToggle}>
+        <TooltipTrigger elementType={Button}>trigger</TooltipTrigger>
+        <TooltipPopover>{popoverText}</TooltipPopover>
+      </TooltipModern>,
+    );
+    const element = dom.container.querySelector('.Tooltip') as HTMLElement;
+
+    expect(element).not.toHaveClass('is-hidden');
+  });
+
+  it('should call toggle function', () => {
+    const onToggle = jest.fn();
+
+    const dom = render(
+      <TooltipModern id={id} isOpen={false} onToggle={onToggle}>
+        <TooltipTrigger elementType={Button}>trigger</TooltipTrigger>
+        <TooltipPopover>Hello World</TooltipPopover>
+      </TooltipModern>,
+    );
+    const trigger = dom.container.querySelector(`button#${id}`) as HTMLElement;
+
+    fireEvent.click(trigger);
+
+    expect(onToggle).toHaveBeenCalled();
+  });
+});
+
+describe('TooltipTrigger', () => {
```

Comment:
Can we place the tests of specific components in the corresponding test files, please?